filebeat/module/nginx/access/machine_learning/datafeed_response_code.json

```diff
@@ -0,0 +1,44 @@
+{
+    "job_id": "JOB_ID",
```

Comment:
Yeah, it gets replaced in code with a generated job id. Didn't know how to reflect this otherwise.